src/SFA.DAS.TeachInFurtherEducation.Web/Startup.cs

```diff
@@ -39,13 +47,13 @@ public Startup(IConfiguration configuration, IWebHostEnvironment env)
             Configuration = new ConfigurationBuilder()
                 .AddConfiguration(configuration)
 
-                .AddAzureTableStorage(options =>
-                {
-                    options.ConfigurationKeys = configuration["ConfigNames"]?.Split(",");
-                    options.StorageConnectionString = configuration["ConfigurationStorageConnectionString"];
-                    options.EnvironmentName = configuration["EnvironmentName"];
-                    options.PreFixConfigurationKeys = false;
-                })
+                //.AddAzureTableStorage(options =>
```

Comment:
I hope, this configuration needs to be uncommented

Reply:
yes you are right, it does

Reply:
we should have a better way of doing that really